src/app.tsx

```diff
@@ -148,7 +150,7 @@ export function App(props: AppProps) {
             >
               {/* Changing the src of iframes will muck up your history. Using key to rerender when the nav changes is a workaround */}
               <iframe
-                key={activeNavItem?.url + uniqueId()}
```

Comment:
This was causing re-render issues. In talking with Joshua, pulling this out for now should mostly be fine but there may be some route issues down the road we need to account for. For the time being though, this seems to be working as expected.

Reply:
I was noticing some annoying re-renders in my [arbitrary-screen-size](https://github.com/vigetlabs/parts-kit/tree/jp/arbitrary-screen-size) branch.  

Hopefully this resolves that issue too